### PR TITLE
model update blueant

### PR DESCRIPTION
**Pull request by erichesse**
# Changes
 - model update:  blueant

### CODE REVIEW

File: assets/mappings/__final__/blueant/activity.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      setValues:
+      - https://mex.rki.de/item/activity-type-3
+    - forValues:
+      - 07 Survey
+      setValues:
+      - https://mex.rki.de/item/activity-type-6
+contact:
+  - fieldInPrimarySource: projectLeaderId (personnelNumber)
+    examplesInPrimarySource:
+    - "projectLeaderId: 20367, personnelNumber: 48217"
```
Comment (mr-kamran-ali) [inline assets/mappings/__final__/blueant/activity.yaml:30]: Can you make sure these are not the real ids from primary source? or just use 12345 to be safe. same for `funderOrCommissioner`, `involvedPerson`, `responsibleUnit`

**Review by mr-kamran-ali — COMMENTED** [review]
nothing major. just making sure if test data is clean.

File: assets/mappings/__final__/blueant/activity.yaml (hunk 1 above)
Comment (erichesse) [reply assets/mappings/__final__/blueant/activity.yaml]: https://github.com/robert-koch-institut/mex-extractors/pull/196/commits/9a0c3f493ea3b233b56376d4d954397ce45bf983

**Review by mr-kamran-ali — APPROVED** [review]
🚀

**Pull request merged**